### PR TITLE
update `pandocfilter` to version `1.5.0`

### PR DESCRIPTION
**Pull request by varlackc**
`pandocfilters` version `1.5.0`
1. - [x] check the upstream
    https://github.com/jgm/pandocfilters/tree/1.5.0

2. - [x] check the pinnings

https://github.com/jgm/pandocfilters/blob/1.5.0/setup.py

https://github.com/jgm/pandocfilters/blob/1.5.0/pandocfilters.py

3. - [x] check the changelogs
    https://github.com/jgm/pandocfilters/tree/1.5.0/CHANGELOG.md

    this will be the last version supporting python 2

4. - [x] additional research
    https://github.com/conda-forge/pandocfilters-feedstock/issues

    - there are currently no open issue at the time of the review on `conda-forge`

5. - [x] verify dev_url
    https://github.com/jgm/pandocfilters

6. - [x] verify doc_url
      - https://pandoc.org/filters.html
7. - [x] license is spdx compliant
      - BSD-3-Clause
8. - [x] license family
      - BSD
9. - [x] verify the build number
      - the build number is set to zero
10. - [x] verify setuptools
      - setuptools is included in the recipe
11. - [x] verify wheel
      - wheel is in the recipe
12. - [x] pip in the test section
13. - [x] veriy the test section

Results:
-

Based on the research findings and the results we can conclude
that it is safe to update `pandocfilters` to version `1.5.0`

### CODE REVIEW

**Review by ScottBailey — APPROVED** [review]
lgtm

**Comment by tobijk** [comment]
lgtm

**Review by tobijk — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,14 +23,15 @@ requirements:
...
+    # The package is not supported for the versions 3.0 to 3.3 of python
+    # https://github.com/jgm/pandocfilters/blob/master/setup.py#L19
+    - python !=3.0,!=3.1,!=3.2,!=3.3
 
 test:
   imports:
     - pandocfilters
   requires:
     - pip
-    - python <3.10
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: If we use noarch, it's better to use `- python <3.10` in `test/requires` as for now we aren't building on python 3.10

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (varlackc) [reply recipe/meta.yaml]: > If we use noarch, it's better to use `- python <3.10` in `test/requires` as for now we aren't building on python 3.10

The restrictions for python 3.10 were added
https://github.com/AnacondaRecipes/pandocfilters-feedstock/blob/update_to_1.5.0/recipe/meta.yaml#L35

**Pull request merged**